### ISSUE TITLE
Navigating to the AI preferences does not work

### ISSUE THREAD

**Issue by koppor** [InAnYan/jabref#122]
I click on "Click here ..." - and nothing happens.

![image](https://github.com/user-attachments/assets/6015898b-42bc-4f74-a096-ed1b4717b251)

The final dot ismissing or is it just that I don't see it at this color contrast?

**Comment by InAnYan** [comment]
It was even worse. It's not an internationalized string :)

Fixed
